packages/network/src/connection/webrtc/WebRtcConnection.ts

```diff
@@ -7,11 +7,18 @@ import { MessageQueue, QueueItem } from '../MessageQueue'
 import { NameDirectory } from '../../NameDirectory'
 import crypto from 'crypto'
 
+export interface IceServer {
+    url: string
+    port: number
```

Comment:
Would it make sense that we'd remove port field? It is a component of a URL string, like e.g. domain and path are.

(Yes, username and credentials are also officially part of URL definition... But they don't specify the _location_ of the server)

Reply:
Yeah that would work. We just need a reliable way of extracting port from url as node-datachannel library requires this.